### DISCUSSION TITLE
How Long to Retain Machine Readable Files

### DISCUSSION THREAD

**Discussion by AJLgeHG** [CMSgov/price-transparency-guide#238] [Q&A] [answered]
Hello - Is there a set amount of time that health plans are required to retain the Machine Readable Files (MRFs) generated monthly, for display on their public website?

I have heard that 10 years is the required amount of time. Is this correct or is the required retention time different?

Thanks.

**Comment by jonberkery-work** [comment]
I am under the impression, there are no retention requirements in the rule and that entities are expected to follow their own corporate retention policy guidelines for this data.  I do not expect to keep this data for more then one month, I am building a process to rebuild the data as of a "date" submitted by a requestor (i.e. - an audit request or legal request).

**Comment by Devyandu** [comment]
I have not seem anything in the regulation which mandates retention for a certain period.
Few factors that might influence your decision:
- Cost of storage
-  Corporate retention guidelines
- Support needs, if you were reported a problem in an older file, you might want a copy to persist for a while.

 Hope this helps
Dev
PS : I am not with CMS. This is my opinion.

**Comment by AJLgeHG** [comment]
Thanks for your responses. We're thinking we'd use our corporate retention policy as a guide for these files, but we're fearing that with the size of the files we will incur exorbitant file storage costs. 

I wasn't able to find any information on this in the regulations. 

Could someone from CMS provide some guidance here?

**Reply by mjbriggs** [reply]
@shaselton I believe this could be in your wheelhouse

**Reply by AJLgeHG** [reply]
@shaselton could you provide a response on this topic?

**Comment by shaselton-usds** [comment, marked as answer]
Data retention can be found here: #69
